src/util/iterative_restoration.jl

```diff
@@ -157,7 +220,7 @@ end
 
 
 "Merge solution dictionaries and accumulate solvetime and objective"
-function merge_solution!(solution1, solution2)
+function _merge_solution!(solution1, solution2)
```

Comment:
If you intend this to specific to iterative restoration a name like `_merge_iterative_solution!` might help avoid future name clashes.  But if you think it is more general and could apply to other meta algorithms then `_merge_solution!` makes sense.

Reply:
I think I will make it more specific for now.  Maybe it could be useful for other meta algorithms in the future, but maybe more control over how networks are merged and MOI  solution status is needed.